assets/component-facets.css

```diff
@@ -903,3 +932,164 @@ input.mobile-facets__checkbox {
 .product-count__text.loading + .loading-overlay__spinner {
   display: block;
 }
+
+@media screen and (min-width: 750px) {
+  .facets-vertical {
+    display: flex;
+  }
+
+  .facets-vertical .disclosure-has-popup[open] > summary::before {
+    content: none;
+  }
+  
+  .facets-vertical .facets__display {
```

Comment:
I see what you mean 🤔 I would still keep them in the same file though since there are still some classes that are shared.

From what I have seen most of the updates are spacing and the direction of classes that is why i kept most of it. And these are examples of some that are re-used: `.facets`, `..facets__summary`, `.facets__disclosure`, `facets__header`, `facets__heading`, `.active-facets__button` and `active-facets__button-inner` at a first glance.

I can separate it or duplicate the classes. I was just trying to re-use as much as I could.

What do you think? 🤔 Happy to go either way

Reply:
Might be best to just flag this as a future refactor to-do

Reply:
Sounds good! Issue created here https://github.com/Shopify/dawn/issues/1527